presto-mongodb/pom.xml

```diff
@@ -14,8 +14,8 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <mongo-java.version>3.6.0</mongo-java.version>
-        <mongo-server.version>1.5.0</mongo-server.version>
-        <netty.version>4.0.32.Final</netty.version>
+        <mongo-server.version>1.22.0</mongo-server.version>
```

Comment:
curious, is mongo version change related to case insensitive collection names?

Reply:
Nope, without updating the version will work also.

Reply:
@tchunwei Let's revert the change of pom.xml so that we can focus on the actual issue. Also, could you resolve conflicts?

Reply:
have reverted the version and pull latest master in

Reply:
Thanks, please revert the change for `org.slf4j` too.

Reply:
@ebyhr sorry I missed that out, has reverted it too.